tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -349,6 +349,15 @@ public function testDirtyOnEnumArrayObject()
         $this->assertTrue($model->isDirty('asEnumArrayObjectAttribute'));
     }
 
+    /**
+     * @requires PHP >= 8.1
+     */
```

Comment:
PHP 8.1 is the minimum version so this isn't needed.

Reply:
I removed the rest of these in https://github.com/laravel/framework/pull/46993

Reply:
Merged your changes & removed unecessary checks in latest commit